nextflow/rmatsTurbo.nf

```diff
@@ -0,0 +1,232 @@
+nextflow.enable.dsl=2
+
+
+process rmats_prep_g1 {
+  tag "rmats_prep_g1"
+  label "mid_memory"
+  publishDir path: "${params.publishDir}/read_outcomes", mode: 'copy', pattern: "prep_*_read_outcomes_by_bam.txt"
+
+  input:
+    tuple val(bam_name), file(bam)
+    each file(gtf)
```

Comment:
Seems like `each` would run this once for each `gtf`, but there should only be a single `gtf` for the workflow: https://www.nextflow.io/docs/latest/process.html#inputs

Reply:
This is correct.
`each` means, for each bam file, nextflow will link gtf each time.


If no `each` here, only one bam file will be processed.